src/components/CompetitionPage/CompetitionPage.tsx

```diff
@@ -99,36 +108,33 @@ export const CompetitionPage: FC<CompetitionPageProps> = ({
           </Typography>
         )}
       </Stack>
-
       <Stack>
         <Typography variant="h2">Archív</Typography>
         {/* TODO: asi zjednotit styly, neriesit with/without publications */}
-        {competition_type?.name === 'Tábor' ? (
-          <Stack gap={1}>
-            {history_events.map((event) => (
-              <Typography key={event.id} variant="h3" component="span">
-                {name + ' '} {event.school_year}
+
+        <Stack gap={1}>
+          {history_events.map((event) => (
+            <Stack
+              key={event.id}
+              direction="row"
+              sx={{
+                justifyContent: 'space-between',
+                alignItems: 'end',
```

Comment:
radsej end
<img width="411" alt="image" src="https://github.com/ZdruzenieSTROM/webstrom-frontend/assets/6044119/5cf6c1c2-92aa-4227-8957-4799713b1d4e">

start (nic):
<img width="410" alt="image" src="https://github.com/ZdruzenieSTROM/webstrom-frontend/assets/6044119/5594cd20-06eb-45b8-b287-1d0e8f5c5fe0">

alebo center?:
<img width="404" alt="image" src="https://github.com/ZdruzenieSTROM/webstrom-frontend/assets/6044119/7af22814-9b4a-434e-942b-8ae878e66415">

sam neviem, ale nejakou klasikou by som tu mozno siel na center...

Reply:
Podla dorot ma byt start tak som to použil